tests/core.py

```diff
@@ -1163,6 +1163,22 @@ def test_fetch_task_instance(self):
         response = self.app.get(url)
         assert "runme_0" in response.data.decode('utf-8')
 
+    def test_task_info(self):
```

Comment:
I think the test(s) should now be moved to

www/api/v0/endpoints.py

Blueprints are essentially plugins and not part of core.